src/server/gc_worker/gc_worker.rs

```diff
@@ -817,6 +818,45 @@ where
                 // Warn and ignore it.
                 warn!("unsafe destroy range: failed sending ClearRegionSizeInRange"; "err" => ?e);
             });
+        } else {
+            let cfs = &[CF_LOCK, CF_DEFAULT, CF_WRITE];
+            let keys = vec![start_key.clone(), end_key.clone()];
+            let regions = get_regions_for_gc(self.store_id, &keys, regions_provider)?;
+            let count = regions.len();
+
+            let mut region_modifies = HashMap::default();
+            for i in 0..count {
+                let range_start = if i == 0 {
+                    start_key.clone()
+                } else {
+                    Key::from_raw(regions[i].get_start_key())
+                };
+                let range_end = if i == count - 1 {
+                    end_key.clone()
+                } else {
+                    Key::from_raw(regions[i].get_end_key())
+                };
+
+                let mut modifies = Vec::new();
+                for cf in cfs {
+                    modifies.push(Modify::DeleteRange(
```

Comment:
cc @BusyJay 
Is it possible to delete large ranges in a safe way (instead of bypassing raft) after supporting multi-rocks?

Reply:
Yes, just send them like normal commands.